frontend/src/routes/FacilityBooking/ViewFacility/index.tsx

```diff
@@ -66,14 +66,23 @@ export default function ViewFacility() {
   }
 
   const onDateClick = (date: Date) => {
-    const dateRowStartDate = getDateRowStartDate(date)
+    if (parseInt(params.isEndDate) === 0) {
```

Comment:
oh are you trying to use params to pass the dateRowStartDate value. In that case remove these and just update yr branch with my fix, which i merged into `revamp/facilites` alr!